src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -122,4 +135,14 @@ private function generateProxyClass(Definition $definition)
 
         return $generatedClass;
     }
+
+    /**
+     * @param Definition $definition
```

Comment:
I would still suggest to drop the docblock entirely :)

Reply:
At least it provides the return value :)

Reply:
so let's keep only the return value if you want... :)

Reply:
done